unity/Holo/Assets/Scripts/ModelWithPlate.cs

```diff
@@ -143,6 +148,7 @@ private void LoadInstance(string newInstancePath, bool newIsPreview)
         instanceTransformation.transform.parent = transform;
 
         instance = Instantiate<GameObject>(template, instanceTransformation.transform);
+        instance.name = "mainModel";
 

```

Comment:
This should not be necessary, following my above comment -- you should not access stuff by name :)

Avoiding this will also keep us ready for the use-case "multilple models in scene", when "mainModel" would not be unique.